src/interfaces/idleUser.type.ts

```diff
@@ -0,0 +1,22 @@
+export type IdleUserArray = {
+  heading: string;
+  content: IdleUser[];
+  error: string | null;
+  isLoading: boolean;
+};
+
+export type IdleUser = {
+  username: string
+  id: string;
+  picture: {
+    url: string;
+  };
+  currentStatus: {
+    state: string;
+    updatedAt: string;
+    from: string;
+    until: string;
+    message: string;
+  };
+  full_name: string;
+};
```

Comment:
The `IdleUser` can also have `monthlyHours` field. something of the sort 
```js
{
  currentStatus: {
    until: 1669256009000,
    message: "Bad Health",
    state: "OOO",
    updatedAt: 1668215609000,
    from: 1668215609000,
  },
  monthlyHours: {
    updatedAt: 1668215609000,
    committed: 40,
  },
``` 
will this interface work for this json ?

Note:  some of the documents may not have the `currentStatus` field as well. Have you handled that edge case.

Reply:
I am unable to understand the question

Reply:
I guess @heyrandhir is trying to say that, sorting can be based on `monthlyHours` too but that can be separately handled I think in a separate PR.

Reply:
No No I meant that the data which we get on calling the api apart from having the keys `username `, `id`, `picture`,`currentStatus` , `full_name` can also have the key `monthlyHours`. I was asking that will this interface work good if we have `monthlyHours` key present in the json. Let me know if its still unclear

Reply:
in our case, as we are not using monthly hours anywhere, we can remove it from the interface. If in future we do use it, we can modify the interface accordingly

Reply:
I get your point we are not using it currently but if say tomorrow in some of the responses we start getting `monthlyHours` this code wouldn't break. Right ?